### PR TITLE
Widget.css_class_name

### PR DESCRIPTION
**Pull request by almarklein**
Implement #213 

The name "css_class_name" is a bit long, but I fear that "class_name" will confuse people. Maybe "css_name", or "style_name"? I'd also consider renaming "style" to sorta group these two props together.

### CODE REVIEW

**Comment by Korijn** [comment]
Do you think this might be a nice grouping?

* `my_widget.css.class_name`
* `my_widget.css.style`

**Comment by Korijn** [comment]
Ironically though, they're just as much HTML attributes as some other Flexx props like `flex`.

**Comment by almarklein** [comment]
I am not sure about the `css`  sub object. It would need another `Model` object to make work nice. The name(s) should be such that it does not confuse Pythonista's who don't know much about CSS/HTML, while web dev can guess that it might be related to the css class name.

E.g. `style_name`, `css_name`, `style_class` , `css_class`.

I think I like `style_class`

**Comment by Korijn** [comment]
I see your point, however, `class_name` or `css_class_name` seems most preferable to me. It's a class name, after all.

**Comment by almarklein** [comment]
I can see how adding the word "class" will help web devs a lot to see what its for. To prevent Pythonistas getting confused we should prefix it with `style` or `css`, IMO. The suffic `name` is not that important, I think, e.g. you also write `<div class='xx'>` and not `<div class_name='x'>`.

**Comment by Korijn** [comment]
`css_class` then?

File: flexx/ui/examples/mondriaan.py [hunk 1]
```diff
@@ -33,25 +39,25 @@ def init(self):
...
                 
                 with MyHBox(flex=2):
-                    ui.Widget(flex=6, style='background:#fff;')
+                    ui.Widget(flex=6, css_class_name='white')
                     
                     with MyVBox(flex=1):
-                        ui.Widget(flex=1, style='background:#fff;')
-                        ui.Widget(flex=1, style='background:#ff7;')
+                        ui.Widget(flex=1, css_class_name='white')
+                        ui.Widget(flex=1, css_class_name='yellow')
```
Comment (Korijn) [inline flexx/ui/examples/mondriaan.py]: This is the first time I've seen [presentational class names](https://seesparkbox.com/foundry/naming_css_stuff_is_really_hard) that are simultaneously also semantic. :)

**Pull request merged**